Multisig/UI/Transactions/TransactionDetails/TransactionDetailsView.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  TransactionDetailsView.swift
+//  Multisig
+//
+//  Created by Moaaz on 5/29/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct TransactionDetailsView: View {
+    @FetchRequest(fetchRequest: Safe.fetchRequest().selected())
+    var selectedSafe: FetchedResults<Safe>
+
+    let transaction: BaseTransactionViewModel
+    var body: some View {
+        List {
+            TransactionHeaderView(transaction: transaction)
+            if !(transaction.confirmations?.isEmpty ?? true) {
+                TransactionConfirmationsView(transaction: transaction, safe: selectedSafe.first!)
```

Comment:
if the selection suddenly changes to nil while this screen is opened, the app will crash

Reply:
Then there will be no transactions to be displayed right?

Reply:
I was thinking about a situation when this screen is already open, and then because of push notification the app switches to a new safe, and then let's say there's a bug... now I see that it becomes very unlikely, but still. At your judgement. 😄